src/sas/qtgui/Utilities/Preferences/PreferencesWidget.py

```diff
@@ -66,6 +66,12 @@ def _stageChange(self, key: str, value: ConfigType):
             message = self.restart_params.get(key, None)
             self.parent.stageSingleChange(key, value, message)
 
+    def _unStageChange(self, key: str):
+        """ A private class method to unstage a single configuration change. Typically when the value is not valid. """
+        if self.parent is not None and hasattr(self.parent, 'unStageSingleChange'):
```

Comment:
just a suggestion.
self.parent can only be None when you're testing the widget, right? maybe instead you could mock the parent in the test and avoid explicit conditional here?